profile/README.md

```diff
@@ -0,0 +1,40 @@
+![Unleash_GitHub_Banner](https://user-images.githubusercontent.com/18158596/196736157-b20a5fbf-74ac-47cd-a991-fe6e2d2bc960.png)
+
+<p align="center">
+<b>The Open Source Feature Flagging Service</b>
+</p>
+
+![Unleash_GitHub_Mocks (2)](https://user-images.githubusercontent.com/18158596/196740001-569c6951-e989-4f7e-b31b-def5c40fa2aa.png)
+
+
+<!-- <img width="600" alt="Unleash Logo" src="https://user-images.githubusercontent.com/18158596/196730085-d168ff9e-8b1a-4790-8219-882336df6479.png"> -->
+
+**Unleash started as a side project.** We wanted to fill an essential space in the development stack and control releases, and, in the beginning, we were just a few contributors. Together we built a foundation for the amazing community that you see today. 🫂
+
+We love that our community is able to work with our product with all the freedom you could ever want. When you can just test, experiment, and release confidently, you'll end up with a well-rounded product with a ton of awesome interpretations. We're proud of you guys. 🙏
+
+That's enough about us. Have fun. Be kind. Let's **Unleash** great software together. 💪
+
+> [📈 See our roadmap →](https://github.com/orgs/Unleash/projects/5)
+
+## 🚀 Learn about Unleash
+Create with Freedom, Release With Confidence.
+
+[Main Open Source Repo](https://github.com/Unleash/unleash) - Unleash's Open Source codebase. 🖥️ </br>
+[Docker Hub](https://hub.docker.com/r/unleashorg/unleash-server) - Unleash containerized, just for you. 🎛️ </br>
+[Unleash Docs](https://docs.getunleash.io) - Dive deeper into the docs. 📚 </br>
+[How-to Guides](https://docs.getunleash.io/how-to) - All the how-to's answered for you. 🙋 </br>
+[Open Live Demo](https://www.getunleash.io/interactive-demo) - Unleash Official demo application using Unleash + Next.js.</br>
+[Feature Flagging - Why and How to Add to Your Software](https://youtu.be/-yHZ9uLVSp4) - Need something visual to get started? Here's something to watch. 📹 </br>
+
+## ✨ Connect with Us</br>
+Get involved with a vibrant community of users and contributors. Check out what others have built with Unleash, start learning by going through Unleash How-to Guides or get involved as a contributor.
+
+[Leave us a Star](https://github.com/Unleash/unleash) on the Unleash Repo. 🌟 </br>
+[GitHub Discussions](https://github.com/Unleash/unleash/discussions) - Place to discuss, ask questions and find answers, show your Unleash project and get feedback. 🗣️ </br>
+[Slack](https://unleash-community.slack.com/) -  Come chat with the Unleash community including the core team. 💬 </br>
+[Community Homepage](https://www.getunleash.io/unleash-community) - Check out the ways you can be a valuable member of the community. 🛰️ </br>
+<!-- [Events at Unleash]() - Learn more about new Events for the Unleash Community. 🎪 </br> -->
+Contributing Guidelines coming soon. ⚠️ </br>
```

Comment:
Any reason you're using regular sentences instead of a list here? It looks like a list to me, so I think the right way to do this would be to use a list. It would also be helpful for people using assistive tech to read this, so that it's correctly called out as a list 💁🏼 

I'd usually also remove comments, but this seems more like a placeholder for now?

```suggestion
- [Main Open Source Repo](https://github.com/Unleash/unleash) - Unleash's Open Source codebase. 🖥️
- [Docker Hub](https://hub.docker.com/r/unleashorg/unleash-server) - Unleash containerized, just for you. 🎛️ 
- [Unleash Docs](https://docs.getunleash.io) - Dive deeper into the docs. 📚
- [How-to Guides](https://docs.getunleash.io/how-to) - All the how-to's answered for you. 🙋
- [Open Live Demo](https://www.getunleash.io/interactive-demo) - Unleash Official demo application using Unleash + Next.js
- [Feature Flagging - Why and How to Add to Your Software](https://youtu.be/-yHZ9uLVSp4) - Need something visual to get started? Here's something to watch. 📹

## ✨ Connect with Us

Get involved with a vibrant community of users and contributors. Check out what others have built with Unleash, start learning by going through Unleash How-to Guides or get involved as a contributor.

- [Leave us a Star](https://github.com/Unleash/unleash) on the Unleash Repo. 🌟
- [GitHub Discussions](https://github.com/Unleash/unleash/discussions) - Place to discuss, ask questions and find answers, show your Unleash project and get feedback. 🗣️
- [Slack](https://unleash-community.slack.com/) -  Come chat with the Unleash community including the core team. 💬 
- [Community Homepage](https://www.getunleash.io/unleash-community) - Check out the ways you can be a valuable member of the community. 🛰️ 
<!-- - [Events at Unleash]() - Learn more about new Events for the Unleash Community. 🎪 -->
- Contributing Guidelines coming soon. ⚠️
```

Reply:
I don't have a strong preference here, to be honest. Let's move forward with lists.

The comment is a placeholder, yes!